examples/fusionauth/environment.external-postgres.yml

```diff
@@ -0,0 +1,25 @@
+interfaces:
+  auth: ${ components['architect/fusionauth'].interfaces.http.url }
+
+components:
+  architect/fusionauth:
+    extends: file:./architect.yml
+    parameters:
+      DATABASE_ROOT_USER: fusionuser
+      DATABASE_ROOT_PASSWORD: fusionpassword
+      DATABASE_USER: fusionuser
+      DATABASE_PASSWORD: fusionpassword
+
+    services:
+      db:
+        interfaces:
+          main:
+            host: host.docker.internal
+            protocol: postgresql
+            port: 5432
+
+      fusionapi:
+        volumes:
+          config:
+            host_path: ./fa_config
```

Comment:
This could be defined in the debug block for the service

Reply:
The config file (and thus mounted volume) is needed for all deploys, local and remote, so it doesn't belong in the debug block. Created ticket: https://gitlab.com/architect-io/architect-cli/-/issues/116